src/components/ClickableMap/ClickableMap.js

```diff
@@ -198,50 +205,106 @@ function ClickableMap({ onLocationChange, locationInfo }) {
     }
   }
 
+  function handlePlaceChange(address) {
+    setCurrentPlaceLabel(address);
+  }
+
+  function handlePlaceSelect(_event, selection) {
+    if (!selection) return;
+
+    geocodeByAddress(selection.description)
+      .then((results) => getLatLng(results[0]))
+      .then((latLng) => {
+        map.setZoom(15);
+        setLocation({ latitude: latLng.lat, longitude: latLng.lng });
+      })
+      .catch((error) => {
+        showError('Failed to get the location from address.');
+        // eslint-disable-next-line no-console
+        console.error('Error', error);
```

Comment:
We could add a `Sentry.captureException` to this error case so the error goes to Sentry